content/Mobile_Dev/WebAuthn/IOS/Authenticate_with_Username.adoc

```diff
@@ -0,0 +1,78 @@
+= Authenticate with Username on IOS Safari
+
+== I want my users to authenticate by first providing their username
+In this section we are going to explore the standard way of authenticating users using WebAuthn. This approach is commonly referred to as identifier first - the user provides their username, and the relying party will send an authentication challenge with a list of valid credentials belonging to a user. The user will only be able to provide a challenge response if they use one of the credentials with an ID included in the list from the relying party. This flow works well for users who do not have a discoverable credential registered on their device. 
```

Comment:
Consider throwing the keyword 'adaptive mfa' here

Reply:
Added in latest commit